test/e2e/framework/deployment/fixtures.go

```diff
@@ -33,7 +33,7 @@ import (
 
 // UpdateDeploymentWithRetries updates the specified deployment with retries.
 func UpdateDeploymentWithRetries(c clientset.Interface, namespace, name string, applyUpdate testutils.UpdateDeploymentFunc) (*appsv1.Deployment, error) {
-	return testutils.UpdateDeploymentWithRetries(c, namespace, name, applyUpdate, framework.Logf, poll, pollShortTimeout)
+	return testutils.UpdateDeploymentWithRetries(c, namespace, name, applyUpdate, framework.Logf, framework.Poll, framework.PollShortTimeout)
```

Comment:
i don't think i like this coupling between framework sub-packages and the root framework WRT timeouts.
keeping timeouts specific to a package makes more sense even if there are duplicates.

Reply:
sure